chain-spec-generator/src/relay_chain_specs.rs

```diff
@@ -0,0 +1,476 @@
+						x.0.clone(),
+						kusama_session_keys(
+							x.2.clone(),
+							x.3.clone(),
+							x.4.clone(),
+							x.5.clone(),
+							x.6.clone(),
+							x.7.clone(),
+							x.8.clone(),
+						),
+					)
+				})
+				.collect::<Vec<_>>(),
+		},
+		staking: kusama_runtime::StakingConfig {
+			minimum_validator_count: 1,
+			validator_count: initial_authorities.len() as u32,
+			stakers: initial_authorities
+				.iter()
+				.map(|x| (x.0.clone(), x.0.clone(), STASH, kusama_runtime::StakerStatus::Validator))
+				.collect(),
+			invulnerables: initial_authorities.iter().map(|x| x.0.clone()).collect(),
+			force_era: Forcing::NotForcing,
+			slash_reward_fraction: Perbill::from_percent(10),
+			..Default::default()
+		},
+		babe: kusama_runtime::BabeConfig {
+			authorities: Default::default(),
+			epoch_config: Some(kusama_runtime::BABE_GENESIS_EPOCH_CONFIG),
+			..Default::default()
+		},
+		grandpa: Default::default(),
+		im_online: Default::default(),
+		authority_discovery: kusama_runtime::AuthorityDiscoveryConfig {
+			keys: vec![],
+			..Default::default()
+		},
+		claims: kusama_runtime::ClaimsConfig { claims: vec![], vesting: vec![] },
+		vesting: kusama_runtime::VestingConfig { vesting: vec![] },
+		treasury: Default::default(),
+		hrmp: Default::default(),
+		configuration: kusama_runtime::ConfigurationConfig {
+			config: default_parachains_host_configuration(),
+		},
+		paras: Default::default(),
+		xcm_pallet: Default::default(),
+		nomination_pools: Default::default(),
+		nis_counterpart_balances: Default::default(),
+	}
+}
+
+fn polkadot_development_config_genesis(
+	wasm_binary: &[u8],
+) -> polkadot_runtime::RuntimeGenesisConfig {
+	polkadot_testnet_genesis(
+		wasm_binary,
+		vec![get_authority_keys_from_seed("Alice")],
+		get_account_id_from_seed::<sr25519::Public>("Alice"),
+		None,
+	)
+}
+
+fn kusama_development_config_genesis(wasm_binary: &[u8]) -> kusama_runtime::RuntimeGenesisConfig {
+	kusama_testnet_genesis(
+		wasm_binary,
+		vec![get_authority_keys_from_seed("Alice")],
+		get_account_id_from_seed::<sr25519::Public>("Alice"),
+		None,
+	)
+}
+
+/// Polkadot development config (single validator Alice)
+pub fn polkadot_development_config() -> Result<Box<dyn ChainSpec>, String> {
+	let wasm_binary =
+		polkadot_runtime::WASM_BINARY.ok_or("Polkadot development wasm not available")?;
+
+	Ok(Box::new(PolkadotChainSpec::from_genesis(
+		"Polakdot Development",
+		"polkadot_dev",
+		ChainType::Development,
+		move || polkadot_development_config_genesis(wasm_binary),
+		vec![],
+		None,
+		Some(DEFAULT_PROTOCOL_ID),
+		None,
+		Some(polkadot_chain_spec_properties()),
+		Default::default(),
+	)))
+}
+
+/// Kusama development config (single validator Alice)
+pub fn kusama_development_config() -> Result<Box<dyn ChainSpec>, String> {
+	let wasm_binary = kusama_runtime::WASM_BINARY.ok_or("Kusama development wasm not available")?;
+
+	Ok(Box::new(KusamaChainSpec::from_genesis(
+		"Kusama Development",
+		"kusama_dev",
```

Comment:
```suggestion
		"kusama-dev",
```
I think this should be aligned with this:
https://github.com/polkadot-fellows/runtimes/pull/127/files#diff-2502728cf4efbc62c3f43ecd31a04613861dcb894cfc75f5ce2389f28f6eecefR25-R31